backend/src/src-licenseinfo/pom.xml

```diff
@@ -59,6 +71,18 @@
             <artifactId>saxon-dom</artifactId>
             <version>8.7</version>
         </dependency>
+        <dependency>
+            <groupId>org.dom4j</groupId>
+            <artifactId>dom4j</artifactId>
+            <version>2.1.1</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>jaxen</groupId>
+            <artifactId>jaxen</artifactId>
```

Comment:
This dependency is already approved via the following CQ: https://dev.eclipse.org/ipzilla/show_bug.cgi?id=11159